.travis.yml

```diff
@@ -1,5 +1,9 @@
+dist: xenial
```

Comment:
What this for?

Reply:
testing on latest LTS ubuntu distro 16.x instead of default 14.x
won't really matter with our use case
but it thought it is a nice thing to have